package.nls.es.json

```diff
@@ -0,0 +1,144 @@
+{
+    "A new configuration to submit ECL to localhost, hthor": "Una nueva configuración para enviar ECL a localhost, hthor",
+    "Abort Workunit": "Aborte unidad de trabajo",
+    "Abort submit when errors are reported during archive generation": "Anular el envío cuando se informan errores durante la generación del archivo",
+    "Activate": "Activar",
+    "Activate Client Tools": "Activar herramientas del cliente",
+    "Add '-legacy' argument to eclcc": "Agregue el argumento '-legacy' a eclcc",
+    "Additional folders to use when resolving IMPORT statements": "Carpetas adicionales para usar al resolver declaraciones IMPORT",
+    "All workunits": "Todos los unidad de trabjo",
+    "Auto Detect": "Detección automática",
+    "Auto Detect Client Tools": "Detección automática de herramientas del cliente",
+    "Automatically open Workunits on creation": "Abrir unidad de trabajo automáticamente al crearse",
+    "Build flags, to be passed to the eclcc compiler": "Build flags, to be passed to the eclcc compiler",
+    "Bundles": "Paquetes",
+    "Bundles Homepage": "Pagina principal de paquetes",
+    "Check syntax on load": "Verifique la sintaxis al cargar",
+    "Check syntax on save": "Compruebe la sintaxis al guardar",
+    "Clear all previously reported ECL Syntax Check results": "Borrar todos los resultados de la comprobación de sintaxis ECL informados anteriormente",
+    "Client Tools": "Herramientas del cliente",
+    "Client Tools Homepage": "Herramientas del cliente pagina principal",
+    "Compile": "Compilar",
+    "Compile ECL on the HPCC Platform": "Compilar ECL en HPCC Systems®",
```

Comment:
Other entries further down refer to the target hpcc platform as "la plataforma HPCC" instead of the proper product name used here "HPCC Systems®"

Reply:
I suspect I need to do a sweep and fix globally.